Server/Api/Startup.cs

```diff
@@ -29,7 +29,7 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-
+            services.AddCors();
```

Comment:
Can you revert these changes? @hakanozcan will handle these changes in startup.cs.